### PR TITLE
[CDF-848] Increase documentation for Dashboard.notifications

### PR DESCRIPTION
**Pull request by afrjorge**

### CODE REVIEW

**Comment by afrjorge** [comment]
@jvelasques @diogofscmariano please review

**Comment by afrjorge** [comment]
@diogofscmariano @jvelasques please review

File: core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js [hunk 1]
```diff
@@ -31,37 +31,53 @@ define([
...
+     * @description Initializes the dashboard notifications.
      *
-     * @param {Object}  options             The options to configure.
-     * @param {string}  options.message     The message or HTML to display in the UI overlay.
-     * @param {Object}  options.css         A json that accepts valid css key/value pairs.
-     * @param {Object}  options.overlayCSS  A json that accepts valid css key/value pairs for the UI overlay.
-     * @param {boolean} options.showOverlay Allows you to show or hide the UI overlay.
+     * @private
+     */
+    _initNotifications: function() {},
```
Comment (diogofscmariano) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js]: are we fine doing this?

Comment (afrjorge) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js]: This module follows the same pattern as all other Dashboard.*.js so I left this private method here. It was only initialising the ERROR_CODES constant during dashboard instantiation. I think it makes more sense to have the constant initialised only one time when the modules are loaded, rather than every time a dashboard is instantiated.
Do you want me to revert this change?

Comment (diogofscmariano) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.notifications.js]: Fine :) lets keep it
